src/java.base/share/classes/sun/security/x509/SubjectKeyIdentifierExtension.java

```diff
@@ -73,7 +73,7 @@ private void encodeThis() throws IOException {
      * @param octetString the octet string identifying the key identifier.
      */
     public SubjectKeyIdentifierExtension(byte[] octetString)
-    throws IOException {
+            throws IOException {
```

Comment:
Do you want to change the `KeyIdentifer` ctor to check for null instead of letting it throw NPE?

Reply:
Ok.